components/org.wso2.carbon.business.rules.core/src/main/java/org/wso2/carbon/business/rules/core/services/BusinessRulesService.java

```diff
@@ -38,16 +38,20 @@
      * @return BusinessRule
      * @throws TemplateManagerServiceException
      */
-    BusinessRule findBusinessRule(String businessRuleUUID) throws TemplateManagerServiceException,
-            BusinessRuleNotFoundException;
+
+    BusinessRule findBusinessRule(String businessRuleUUID)
+            throws TemplateManagerServiceException, BusinessRuleNotFoundException;
 
     /**
      * Deletes the Business Rule that has the given UUID and Undeploys the Templates belonging to the Business Rule
      *
      * @param uuid UUID of the saved Business Rule definition
      */
-    int deleteBusinessRule(String uuid, Boolean forceDeleteEnabled) throws SiddhiAppsApiHelperException,
-            BusinessRuleNotFoundException, TemplateManagerServiceException;
+    int deleteBusinessRule(String uuid, Boolean forceDeleteEnabled)
+            throws SiddhiAppsApiHelperException, BusinessRuleNotFoundException, TemplateManagerServiceException;
+
+    int deployOrUndeployBusinessRule(String businessRulesUUID, boolean shouldUndeploy) throws TemplateInstanceCountViolationException, TemplateManagerServiceException, RuleTemplateScriptException, BusinessRuleNotFoundException;
```

Comment:
Line length exceeds 120 characters. Run checkstyle and fix the issues in all the places.

Reply:
I will check and do the changes